README.md

```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           artefacts-api-key: ${{ secrets.ARTEFACTS_API_KEY }}
           job-name: basic_tests
-          # If you prefer to see the commit name instead of the workflow name.
+          # If you prefer a commit (or custom) name instead "Artefacts CI" in the platform.
```

Comment:
if my understanding is correct, it is more accurate to keep the original for the second part of the change : "of the workflow name"

Reply:
I changed that in my last commit, not it's "Artefacts CI" by default in the platform. The CI will keep the name so it's easier to read what fails or works on GH?

Reply:
Thank you, makes sense